meta-oe/recipes-connectivity/hiba/hiba_git.bb

```diff
@@ -17,26 +17,22 @@ PV = "1.0+git${SRCPV}"
 inherit autotools
 
 DEPENDS = "openssl openssh zlib"
-RDEPENDS:${PN} = "bash"
 
-def get_oeconf(d):
-  import glob
-  workdir = d.getVar("WORKDIR")
-  openssh_paths = glob.glob("{}/../../openssh/*/openssh-*".format(workdir))
-
-  if len(openssh_paths) != 1:
-    # Cannot fail here since this will cause failure on other recipes
-    # if openssh is not installed
-    return ""
-
-  return "--prefix=/usr/ --with-opensshdir={}".format(openssh_paths[0])
-
-EXTRA_OECONF = "${@get_oeconf(d)}"
+EXTRA_OECONF = " \
+  --with-opensshdir=${PKG_CONFIG_SYSROOT_DIR}/usr/include/ssh \
+  --prefix=/usr/ \
+"
 
 # Make sure that MACHINE_FEATURES includes hiba before continuing
 do_configure:prepend(){
-  if ${@bb.utils.contains("MACHINE_FEATURES", "hiba", "false", "true", d)} ; then
-    echo "MACHINE_FEATURES does not support hiba"
+  if ${@bb.utils.contains("DISTRO_FEATURES", "hiba", "false", "true", d)} ; then
+    echo "DISTRO_FEATURES does not support hiba. Skip the configuration"
```

Comment:
Consider something like below for diagnostic message 
`Skip building ${PN} because hiba support is disabled, It can be enabled it via DISTRO_FEATURES`

Reply:
Done.